apps/meteor/server/email/IMAPInterceptor.ts

```diff
@@ -94,14 +103,22 @@ export class IMAPInterceptor extends EventEmitter {
 		});
 	}
 
+	reconnect(): void {
```

Comment:
A suggestion: To avoid infinite retries like Kevin mentioned, maybe we can add logic here only to attempt max X retries - also, we can even keep this value as a setting. I feel a good number should be 10 retries.